projects/dotcms/src/lib/api/DotApiConfiguration.ts

```diff
@@ -18,7 +18,7 @@ export class DotApiConfiguration {
             .request({
                 url: '/api/v1/configuration'
             })
-            .then(async (response: Response) => {
```

Comment:
why do you remove the Response type?

Reply:
Previously `dotCMSHttpClient.request` returned `Promise<any>` which required adding the `Response` type here. Now that `@types/node-fetch` is included, these types can be inferred.